### DISCUSSION TITLE
Win Server 2022

### DISCUSSION THREAD

**Discussion by arielsil** [AveYo/MediaCreationTool.bat#184] [Ideas]
Could we include the https://software-download.microsoft.com/download/sg/20348.169.210806-2348.fe_release_svc_refresh_SERVER_EVAL_x64FRE_en-us.iso 

in the options?

**Comment by AveYo** [comment]
No. It's not even an esd, it's already an iso. Just download it directly and burn it / put it on usb with rufus or something
